cypress/integration/cucumber/step_definitions/login-test.js

```diff
@@ -0,0 +1,37 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+
+/// <reference types="Cypress" />
+
+beforeEach(() => {
+  cy.initiateExceptionsLogger();
+});
+
+afterEach(() => {
+  cy.end();
+});
+
+Given('User opens the website at {string}', (url) => {
+  cy.visit(url);
+});
+
+When('User enters username {string}', (username) => {
+  cy.get('#username').type(username);
+});
+
+And('User clicks {string}', (text) => cy.get('.iw0iHcFjY--gDZi9jUQ1_').click());
+
+And('User enters Password {string}', (password) => {
+  cy.get('.NGQQUK5YhTAwqMeeVQW5R').type(password);
```

Comment:
is `NGQQUK5YhTAwqMeeVQW5R` always the class for the password field?

Reply:
The class name looked bizarre to me. I'll check to find out if it's randomly autogenerated

Reply:
Thank you for pointing this out.